hack/traffic_flow_tests.sh

```diff
@@ -27,4 +27,5 @@ temp_file=$(mktemp)
 
 envsubst < ../hack/cluster-configs/ocp-tft-config.yaml > $temp_file
 
+TFT_MANIFESTS_OVERRIDE=/root/dpu-operator/hack/cluster-configs/ocp_traffic_flow_manifests
```

Comment:
I think this should be a relative path, so that it works with any checkout. Also note how the Jenkins systems tend to checkout `/home/jenkins/workspace/$PROJECT/dpu-operator/`.

Also, I think this needs an `export TFT_MANIFESTS_OVERRIDE=...` to take effect.

Also, the variables is called `TFT_MANIFESTS_OVERRIDES`. The README is wrong, there needs to be a trailing "S" (I will fix the k8s-tft README).

In the logfile of tft, you will see which path is used. Would be good to confirm that it works.